toolkit/tools/imagecustomizerapi/iso.go

```diff
@@ -9,13 +9,24 @@ import (
 
 // Iso defines how the generated iso media should be configured.
 type Iso struct {
-	AdditionalFiles AdditionalFilesMap `yaml:"AdditionalFiles"`
+	KernelCommandLine KernelCommandLine  `yaml:"KernelCommandLine"`
```

Comment:
You don't seem to be supporting other parts of the `KernelCommandLine` type. So, you probably shouldn't be using the `KernelCommandLine` type here.

Feel free to place `ExtraCommandLine` directly under `Iso`. (Perhaps rename to `ExtraKernelCommandLine`?)

Reply:
Placed it directly...
Can selilnux be supported for the iso file system though? If so, to not break schema compatibility, we'd need to use the `KernelCommandLine` and error out if somebody tries to use selinux while it is not implemented yet.

Reply:
Decided to report an error when selinux is specified under the iso configuration. That way, we avoid creating a new type, and then in the future, should we enable selinux for iso image, we have a consistent kernel schema.